docs/scss/docs.scss

```diff
@@ -38,13 +38,24 @@
   padding-bottom: 100px;
 }
 
+$color-github: #333;
```

Comment:
Didn't do it by purpose to show that this color is chosen because it's GutHubs logo color instead of a random color from our palette.